deployment/v3/mosip/ida/README.md

```diff
@@ -4,5 +4,10 @@
 ```
 ./install.sh
 ```
+* During the execution of the `install.sh` script, a prompt will appear, requesting information regarding the presence of a public domain and a valid SSL certificate on the server.
+If the server lacks a public domain and a valid SSL certificate, it is advisable to select the `n` option. Opting for this choice will enable the init-container with an `emptyDir` volume and include it in the deployment process.
```

Comment:
Opting it will enable the `init-container` with an `emptyDir` volume and include it in the deployment process.